go.sum

```diff
@@ -57,6 +57,7 @@ github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32 h1:Mn26/9ZMNWSw9C9ER
 github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32/go.mod h1:GIjDIg/heH5DOkXY3YJ/wNhfHsQHoXGjl8G8amsYQ1I=
 github.com/giantswarm/apiextensions v0.0.0-20190904125954-a1822dc1393b h1:FQLyt0dyXSvZ9ZvC/E4dDDYi4hbB7C1QUzLujJV2wiM=
 github.com/giantswarm/apiextensions v0.0.0-20190904125954-a1822dc1393b/go.mod h1:zHLSgCszRMdZUzwmfOibzcyfRC89ygwAQlpXcukgZuc=
+github.com/giantswarm/apiextensions v0.0.0-20191004090949-c43a075b8e7d h1:sd3A3ZwXwl3j3KbgKulWLV7KGTQmK8zt9GvR5ALiEE4=
```

Comment:
We need to figure out where this module is coming from.  `v0.0.0-201910` means it's pulling in a version from this month. We should be pinned to Kubernetes 1.13.2 (or maybe 1.15.4 as of yesterday).